Documentation/building/facade-code-coverage.bat

```diff
@@ -30,21 +30,36 @@ SET originalfolder=%cd%
 SET sourcefolder=%corefx%\src\%project%\tests
 
 SET coreclrbuild=%coreclr%\bin\Product\Windows_NT.x64.Debug
+SET coreclrbuild=%coreclr%\bin\Product\Windows_NT.x64.Release
 
 :: Build the library
 cd %sourcefolder%
 msbuild %msbuildargs%
 cd %originalfolder%
 
 :: Delete old files (see #8381 for why)
-del %targetdir%\coreclr.dll
 del %targetdir%\mscorlib.dll
 del %targetdir%\mscorlib.ni.dll
+del %targetdir%\System.Private.CoreLib.dll
+del %targetdir%\System.Private.CoreLib.ni.dll
+del %targetdir%\coreclr.dll
+del %targetdir%\CoreRun.exe
+del %targetdir%\CoreConsole.exe
+del %targetdir%\clretwrc.dll
+del %targetdir%\clrjit.dll
+del %targetdir%\dbgshim.dll
+del %targetdir%\mscordaccore.dll
+del %targetdir%\mscordbi.dll
+del %targetdir%\mscorrc.debug.dll
+del %targetdir%\mscorrc.dll
+del %targetdir%\sos.dll
```

Comment:
Related question; do all of these need to be deleted, or can we just delete the ones (if any) that won't be overwritten by the copy?